adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/CoverageRDD.scala

```diff
@@ -0,0 +1,225 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bdgenomics.adam.rdd.features
+
+import org.apache.spark.rdd.RDD
+import org.bdgenomics.adam.models.{ SequenceRecord, SequenceDictionary, Coverage, ReferenceRegion }
+import org.bdgenomics.adam.rdd.GenomicRDD
+
+import scala.annotation.tailrec
+
+object CoverageRDD {
+
+  /**
+   * @param elem The coverage element to extract a sequence record from.
+   * @return Gets the SequenceRecord for coverage.
+   */
+  private def getSequenceRecord(elem: Coverage): SequenceRecord = {
+    SequenceRecord(elem.contigName, 1L)
```

Comment:
we are not converting from a FeatureRDD, but from an RDD[Coverage]. FeatureRDD does the same thing when a sequence dictionary is not provided (see https://github.com/bigdatagenomics/adam/blob/810806c64e39ff8964f8ed6e3becd4230abc660b/adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureRDD.scala line 84).